bittide-instances/src/Bittide/Instances/Hitl/Tcl/ExtraProbes.hs

```diff
@@ -0,0 +1,55 @@
+-- SPDX-FileCopyrightText: 2024 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+module Bittide.Instances.Hitl.Tcl.ExtraProbes where
+
+import Clash.Prelude
+
+import Clash.Cores.Xilinx.Dna
+import Bittide.Instances.Domains
+import Clash.Annotations.TH ( makeTopEntity )
+import Clash.Cores.Xilinx.Extra
+import Clash.Cores.Xilinx.VIO
+import Data.Maybe
+
+-- | A circuit that verifies the correct behavior of the TCL infrastructure for
+-- setting extra probes in Hitl tests.
+extraProbesTest ::
```

Comment:
Don't forget to add a `NOINLINE`. GHC might give it a different name if you don't.